### PR TITLE
Fix profile updating

### PR DESCRIPTION
**Pull request by igaponov**
https://github.com/PowerlineApp/powerline-mobile/issues/601
https://github.com/PowerlineApp/powerline-mobile/issues/602

### CODE REVIEW

File: backend/src/Civix/ApiBundle/Security/Core/ApiUserProvider.php [hunk 1]
```diff
@@ -130,7 +130,7 @@ public function loadUserByOAuthUserResponse(UserResponseInterface $response)
             $user->{'set'.ucfirst($property)}($username);
             $user->{'set'.ucfirst($propertySecret)}($response->getTokenSecret());
             if ($response->getProfilePicture()) {
-                $content = $this->converter->convert($response->getProfilePicture());
+                $content = $this->converter->convert((string)$response->getProfilePicture());
```
Comment (austinpapp) [inline backend/src/Civix/ApiBundle/Security/Core/ApiUserProvider.php]: why do we need to cast a string here? what else was it returning?

**Comment by austinpapp** [comment]
How is this related to PowerlineApp/powerline-mobile#601?

**Comment by igaponov** [comment]
@austinpapp frontend sends here all data that it has for user and backend handles `avatar_file_name` as a new avatar. After this change it'll ignore links from powerline-*.imgix.net

File: backend/src/Civix/ApiBundle/Security/Core/ApiUserProvider.php (hunk 1 above)
Comment (igaponov) [inline backend/src/Civix/ApiBundle/Security/Core/ApiUserProvider.php]: null?

**Pull request merged**